cmd/release-controller/info.go

```diff
@@ -417,84 +417,131 @@ cp /tmp/pull-secret/* /tmp/.docker/ || true
 oc registry login
 
 cat <<END >>/tmp/serve.py
-import re, os, subprocess, time, threading, socket, SocketServer, BaseHTTPServer, SimpleHTTPServer
+import re
```

Comment:
completely refactoring this makes this much harder to review.  If you want to restructure the code, do it in a separate commit before making substantive changes.

Reply:
My bad.  I've broken the changes down into more meaningful chunks starting with the refactoring piece and then modifications.